internal/broadcaster/utxo_creator.go

```diff
@@ -147,13 +164,13 @@ func (b *UTXOCreator) CreateUtxos(requestedOutputs int, requestedSatoshisPerOutp
 					delete(satoshiMap, res.Txid)
 				}
 
-				// do not performance test ARC when creating the utxos
```

Comment:
We shouldn't remove such comments as they can facilitate the understanding of the code

Reply:
I thought i pushed the change, somehow it went missing pushed again